### ISSUE TITLE
crashed

### ISSUE THREAD

**Issue by fsavalam** [avocode/avocode#978]
IMPORTANT: Please try to describe how did it happen.

-----
App v.2.8.2
Source UID: 78930
```
Error: ENOENT: no such file or directory, open 'C:\Users\villes\.avocode\userdata\78930\documents\e695299c66524cdfb6cc1c0889a0923b\images\resources.tar.gz'
    at Error (native)
```

**Comment by connor-bean** [comment]
Thanks for the report. Can you download Avocode 2.8.4 for Windows x64 bit [here](https://avocode-media.s3.amazonaws.com/download/avocode-app/2.8.4/avocode-app-windows-x64-zip-2.8.4.zip)?

Please let us know if this fixes the problem or not

**Comment by petrbrzek** [comment]
Yes, this should be fixed in 2.8.4. I'm closing this now.
